analyzers/src/SonarAnalyzer.Common/Helpers/KnownMethods.cs

```diff
@@ -158,6 +158,11 @@ public static bool IsGetObjectData(this IMethodSymbol methodSymbol)
             && methodSymbol.Name == nameof(Debug.Assert)
             && methodSymbol.ContainingType.Is(KnownType.System_Diagnostics_Debug);
 
+        public static bool IsDebugFail(this IMethodSymbol methodSymbol) =>
```

Comment:
This is used just once. And there's `.Is(KnownType, Name)` extension available that we should use.